cluster.go

```diff
@@ -47,32 +47,34 @@ import (
 
 type mongoCluster struct {
 	sync.RWMutex
-	serverSynced  sync.Cond
```

Comment:
This happens a lot in go:  the newest member added 'maxSocketReuseTime' has a longer name than any of the previous method names.  As a result, everything shifts right.